concent_api/concent_api/middleware.py

```diff
@@ -14,3 +18,26 @@ def __call__(self, request):
         response = self.get_response(request)
         response['Concent-Golem-Messages-Version'] = __version__
         return response
+
+
+class ConcentVersionMiddleware(object):
+    """
+    Used to attach version of the Concent currently used to HTTP response header.
+
+    """
+
+    CONCENT_VERSION = None
```

Comment:
This should be `_concent_version` because
- It's not really a constant (it's value is `None` and can stay at it for a non-negligible amount of time - until someone calls `get_concent_version()`)
- From the code below I see that you want the user of the class to get it via `get_concent_version()` and not directly. It should be private.

Reply:
:+1: But about second point please check explanation below. Actually I don't want anyone to use it in other way than through middleware. If we want a reusable function it should be in utils.